README.md

```
@@ -31,26 +35,79 @@ uv sync
+uv run pytest tests/test_k8s.py -v
+
+# Run linting
+uv run ruff check src/
+
+# Format code
+uv run ruff format src/
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)